helm-charts/explorer/templates/explorer-configmap.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-config
+  labels:
+    name: {{ $.Values.db.name }}
+    app.kubernetes.io/name: {{ $.Values.db.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }} 
+data:
+
+  basic-network.json: |-
+      {
+      "name": "basic-network",
+      "version": "1.0.0",
+      "client": {
+        "tlsEnable": true,
+        "adminCredential": {
+          "id": "{{ .Values.explorer.username }}",
+          "password": "{{ .Values.explorer.password }}"
+        },
+        "enableAuthentication": true,
+        "organization": "org1MSP",
+        "connection": {
+          "timeout": {
+            "peer": {
+              "endorser": "300"
+            },
+            "orderer": "300"
+          }
+        }
+      },
+      "channels": {
+        "mychannel": {
+          "peers": {
+            "peer0org1": {}
+          },
+          "connection": {
+            "timeout": {
+              "peer": {
+                "endorser": "6000",
+                "eventHub": "6000",
+                "eventReg": "6000"
+              }
+            }
+          }
+        }
+      },
+      "organizations": {
+        "org1MSP": {
+          "mspid": "org1MSP",
+          "adminPrivateKey": {
+            "pem": "-----BEGIN PRIVATE KEY-----\nMIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgUH2s+pegkyjaef+X\nyYBBP52PANixdkt7+xRgTkVPQPahRANCAATChVh6QCmBtI4/6S93XfEj7EDlz0Tf\nISNDAB3rsyjj3T60QhRGczSQmFyEkx6nrrWNyXLsTe/gr4Cc0l//QKQp\n-----END PRIVATE KEY-----\n"
+
+          },
+          "peers": [
+            "peer0org1"
+          ],
+          "signedCert": {
+            "pem": "-----BEGIN CERTIFICATE-----\nMIICBjCCAaygAwIBAgIRAPW8giTkyghYAbO1LEBPbbQwCgYIKoZIzj0EAwIwWzEL\nMAkGA1UEBhMCVVMxEzARBgNVBAgTCkNhbGlmb3JuaWExFjAUBgNVBAcTDVNhbiBG\ncmFuY2lzY28xDTALBgNVBAoTBG9yZzExEDAOBgNVBAMTB2NhLm9yZzEwHhcNMjMw\nMjI3MDYzNDAwWhcNMzMwMjI0MDYzNDAwWjBfMQswCQYDVQQGEwJVUzETMBEGA1UE\nCBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMNU2FuIEZyYW5jaXNjbzEOMAwGA1UECxMF\nYWRtaW4xEzARBgNVBAMMCkFkbWluQG9yZzEwWTATBgcqhkjOPQIBBggqhkjOPQMB\nBwNCAATChVh6QCmBtI4/6S93XfEj7EDlz0TfISNDAB3rsyjj3T60QhRGczSQmFyE\nkx6nrrWNyXLsTe/gr4Cc0l//QKQpo00wSzAOBgNVHQ8BAf8EBAMCB4AwDAYDVR0T\nAQH/BAIwADArBgNVHSMEJDAigCDwWXLSHcu7lm4j8hzmjRtZV9TmXPv9GQiFJZJC\n2XpJgjAKBggqhkjOPQQDAgNIADBFAiEAu5qp7qKw6xYPl5miUgvNeGgGv6hVl17a\n1e7+DUExnZYCIGNsHEehl+VaRQ9c92vBStek6EgwqJ3dtXn3O7aEnT3f\n-----END CERTIFICATE-----\n"
+          }
```

Comment:
these properties need to be dynamic

Reply:
made above values  as configurable in values.yaml